amplify/backend/function/fetchMotionState/src/utils.js

```diff
@@ -139,6 +138,36 @@ const getStakerReward = async (motionId, userAddress, colonyAddress) => {
   };
 };
 
+const didMotionPass = ({
+  motionStakes: {
+    raw: { yay: yayStakes, nay: nayStakes },
+  },
+  requiredStake,
+  revealedVotes: {
+    raw: { yay: yayVotes, nay: nayVotes },
+  },
+}) => {
+  if (
+    BigNumber.from(nayStakes).gte(requiredStake) &&
+    BigNumber.from(yayStakes).gte(requiredStake)
```

Comment:
We could extract at least the `BigNumber.from(yayStakes)` to its own variable, it happens 3 times in this function